pkg/process/metadata/workloadmeta/grpc_test.go

```diff
@@ -308,3 +333,30 @@ func toEventSet(proc *procutil.Process) *pbgo.ProcessEventSet {
 func toEventUnset(proc *procutil.Process) *pbgo.ProcessEventUnset {
 	return &pbgo.ProcessEventUnset{Pid: proc.Pid}
 }
+
+// makeTestClientServer a test extractor, server, and client connection.
+// Cleanup is handled automatically via T.Cleanup().
+func makeTestClientServer(t *testing.T) (*WorkloadMetaExtractor, *GRPCServer, *grpc.ClientConn, pbgo.ProcessEntityStream_StreamEntitiesClient) {
```

Comment:
[nit] what do you think of renaming it to `setupGRPCTest`?

Reply:
I agree that name sounds much better!